Containers/mastercontainer/apache.conf

```diff
@@ -0,0 +1,106 @@
+StartServers             3
+MinSpareThreads         75
+MaxSpareThreads        250
+ThreadsPerChild         25
+MaxRequestWorkers      400
+MaxConnectionsPerChild   0
+PidFile "/run/apache2/httpd.pid"
+
+ServerTokens Prod
+ServerSignature On
```

Comment:
what does this?

Reply:
![grafik](https://github.com/nextcloud/all-in-one/assets/75573284/517816cd-d16a-4241-88ac-291bd0d3cec6)
![grafik](https://github.com/nextcloud/all-in-one/assets/75573284/c5decc74-3df0-4e66-b3e8-69abc1036075)

Reply:
So do we need it?

Reply:
servertokens yes, the other thing, not sure